features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/templates/repository/conf/output-event-adapters.xml.j2

```diff
@@ -98,6 +98,8 @@
         <property key="mail.smtp.port">{{output_adapter.email.port}}</property>
         <property key="mail.smtp.starttls.enable">{{output_adapter.email.enable_start_tls}}</property>
         <property key="mail.smtp.auth">{{output_adapter.email.enable_authentication}}</property>
+        <property key="mail.smtp.signature">{{output_adapter.email.signature}}</property>
+        <property key="mail.smtp.replyTo">{{output_adapter.email.replyTo}}</property>
```

Comment:
Change the name of the key {{output_adapter.email.reply_to}}

Reply:
done